File: Examples/VisualBasic/AsposePDF/Pages/InsertEmptyPage.vb
Imports System.IO
Imports System
Imports Microsoft.VisualBasic
Imports Aspose.Pdf

Namespace AsposePDF.Pages
    Public Class InsertEmptyPage
        Public Shared Sub Run()
            ' ExStart:InsertEmptyPage
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_AsposePdf_Pages()

            ' Open document
            Dim pdfDocument1 As New Document(dataDir & Convert.ToString("InsertEmptyPage.pdf"))

            ' Insert a empty page in a PDF
            pdfDocument1.Pages.Insert(2)
            dataDir = dataDir & Convert.ToString("InsertEmptyPage_out_.pdf")
            ' Save output file
            pdfDocument1.Save(dataDir)
            ' ExEnd:InsertEmptyPage
            System.Console.WriteLine(Convert.ToString(vbLf & "Empty page inserted successfully." & vbLf & "File saved at ") & dataDir)
        End Sub
    End Class
End Namespace

File: Examples/CSharp/AsposePDF/Pages/InsertEmptyPage.cs
//////////////////////////////////////////////////////////////////////////
// Copyright 2001-2013 Aspose Pty Ltd. All Rights Reserved.
//
// This file is part of Aspose.Pdf. The source code in this file
// is only intended as a supplement to the documentation, and is provided
// "as is", without warranty of any kind, either expressed or implied.
//////////////////////////////////////////////////////////////////////////
using System.IO;

using Aspose.Pdf;

namespace CSharp.AsposePdf.Pages
{
    public class InsertEmptyPage
    {
        public static void Run()
        {
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_AsposePdf_Pages();

            //open document
            Document pdfDocument1 = new Document(dataDir + "InsertEmptyPage.pdf");

            //insert a empty page in a PDF
            pdfDocument1.Pages.Insert(2);

            //save output file
            pdfDocument1.Save(dataDir + "InsertEmptyPage_out.pdf");

        }
    }
}